Documentation update for SCP and SCP-B2B packages (#843)
commerce-docs/microsite-commerce-storefront: src/content/docs/releases/changelog.mdx

**Storefront Compatibility Package** 
************************************/}

      <ChangelogEntry
          date="2026-04-20"
          title="Storefront Compatibility Package v4.8.19"
          components={['Storefront Compatibility Package']}
      >
          The Storefront Compatibility Package has been updated to include the following changes:

>           - **Login as Customer**: Introduced a “Login as Customer” feature to support seller-assisted buying. This allows admins to generate a one-time login code from the customer view page backend and authenticate as the customer via GraphQL APIs. Added `admin_assistance_actions` to the `Customer` type to track admin-specific activities or logs, and `admin_assisted_order` in the `CustomerOrder` type to indicate orders placed by an admin. Also introduced the `exchangeOtpForCustomerToken` mutation to retrieve a customer token using the one-time code, alongside the existing `generateCustomerToken` mutation.

Reviewer: ```suggestion
          - **Login as Customer**: A new "Login as Customer" feature enables seller-assisted buying. Admins can now generate a one-time login code from the customer view page in the backend and use it to authenticate as the customer via GraphQL APIs. Two new fields have been added: `admin_assistance_actions` to the `Customer` type (to track admin activities) and `admin_assisted_order` to the `CustomerOrder` type (to mark orders placed by admins). The new `exchangeOtpForCustomerToken` mutation retrieves a customer token using the one-time code, complementing the existing `generateCustomerToken` mutation.
```

Reviewer 2: Add a link to https://experienceleague.adobe.com/en/docs/commerce-admin/customers/customer-accounts/manage/login-as-customer and to the new mutation.

Rewrite this to mention the new mutation before you list the changes to the `Customer` type.

Author: Hello @keharper 
As I shared, we moved the LAC code from ACCS to SCP because the feature overlapped.
Ref. -
https://git.corp.adobe.com/core-commerce-saas/commerce-core-saas-service/blob/main/application/modules/Magento/OtpGraphQl/etc/schema.graphqls

It was originally introduced in ACCS, but due to the code sync-up, we needed to move it to SCP. Later, the ACCS team will remove it from their codebase as per our earlier discussions.

On Friday, we have another call with the ACCS team regarding the RL sharing and LAC code merge. For more details, you can connect with Taras and Vinod.

>           - **WishList Items**: Resolved an issue with wishlist items in multi-store view scenarios under the same website, wishlist items and items_count are now consistent and properly synchronized.

Reviewer: ```suggestion
          - **WishList Items**: Fixed inconsistencies in wishlist items and item count across multiple store views within the same website. Wishlist data is now properly synchronized.
```

Reviewer 2: No captial L in wishlist

>           - **Cart Items**: Resolved an issue where cart item availability checks were failing for products with "Manage Stock" set to No.

Reviewer: ```suggestion
          - **Cart Items**: Fixed cart availability checks for products where "Manage Stock" is disabled. Cart items now correctly display availability status.
```

Reviewer 2: change "display" to "return"

      </ChangelogEntry>

      <ChangelogEntry
          date="2026-04-17"
          title="Storefront Compatibility Package v4.7.12"
          components={['Storefront Compatibility Package']}
      >
          The Storefront Compatibility Package has been updated to include the following changes:

>           - **Order Cancellation**: Resolved issues in `requestGuestOrderCancel`, `cancelOrder`, and `confirmCancelOrder` mutations related to order comments in the response.

Reviewer: ```suggestion
          - **Order Cancellation**: Fixed issues with order comments in the response for the `requestGuestOrderCancel`, `cancelOrder`, and `confirmCancelOrder` mutations.
```
